docs/workforce-management-framework/topics/con-security-introduction.adoc

```diff
@@ -0,0 +1,49 @@
+[]
+= Introduction to {WFM-RC-NameShort} security features
+
+{WFM-RC-NameShort} server gives developers a choice one of the following authentication and authorisation providers
+link:http://www.keycloak.org/[Keycloak] and link:http://passportjs.org/[Passport.js].
+{WFM-RC-NameShort} is build around security interface, which abstracts from any security libraries.
+Both implementations can be swapped at any time without breaking internal api of {WFM-RC-NameShort} modules and reference applications.
+Users can also provide their own security implementation by implementing their own security interface.s
+
+Another security measures are incorporated into our reference applications.
+Please note that all link:https://expressjs.com/[express] based routes outside WFM modules will also need to use these security interfaces.
+
+== How to apply authorization using security interface
+You can choose Keycloak or Passport.js implementation but you can securing endpoints in the same way.
```

Comment:
This section (lines 14-17) needs to be re-written. Its not very readable or provides anything of value to the end user.

As this is a "How" section, it should probably include some code example of keycloak and passport.

Reply:
rewritten to include code eg.